src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts

```diff
@@ -72,6 +75,14 @@ export class DashboardWidgetWrapper extends AngularDisposable implements OnInit
 		}
 	}
 
+	public get showTitle(): boolean {
```

Comment:
do you really need this property? isn't it related to the collapsed state?

Reply:
yes this is needed if a widget wants to always show the title(when collapsed and expanded), compared to how properties widget only wants to show the title when collapsed.